gh-codeql

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 set -e
 
-if [ "$(gh config get extensions.codeql.debug)" = "true" ] ; then
+debug="$(gh config get extensions.codeql.debug 2> /dev/null)" || :
```

Comment:
Worth a code comment about this pattern of swallowing the error from stderr.

Reply:
Thanks, I've added a comment everywhere I've done this.